### DISCUSSION TITLE
[what a great ideea to write kilometric changelogs in the module description!]

### DISCUSSION THREAD

**Discussion by avataros** [PrestaShop/PrestaShop#31872] [Ideas]
### The problem

[what a great ideea to write kilometric changelogs in the module description!] 
When I have to scroll many many screens nobody ever reads twice just to click on the update module button it is the mos pleasant UI experience. 
It competes successfully with reading Apple's terms and conditions.
Sorry, I feel much better now.

### The solution

Write only a short module description in the short module description field?

### Do you plan to work on this subject?

- [X] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [X] I'm willing to submit a Pull Request that implements this feature.
- [X] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by kpodemski** [comment]
Thanks, @avataros ;) It has been improved by @SharakPL for PrestaShop 9.

**Comment by SharakPL** [comment]
It's not an issue in PS 8.0.2 and updated ps_distributionapiclient module. There's just a _Read more_ link now with all module's details in a modal. And in case someone decides to load changelog or long description later it will be handled properly.
